pkg/auth/handler/sso/auth_handler.go

```diff
@@ -74,19 +73,16 @@ func (p AuthRequestPayload) Validate() error {
 
 // AuthHandler decodes code response and fetch access token from provider.
 //
-// curl http://localhost:3000/sso/<provider>/auth_handler?code=<code>&state=<state>
-//
 // For ux_mode is 'mobile_app',
-// it creates a 302 response, and Location points to:
-// myapp://user.skygear.io/sso/{provider}/auth_handler?result=
+// it returns a 302 response with Location set to
+// <callback_url>?x-skgyear-result=...
 //
 // Fox ux_mode is 'web_redirect',
-// it creates a 302 response, and Location points to: sso_callback_url
-// and set cookie in the response.
+// it returns a 302 response with Location set to
```

Comment:
Seems there is no difference between two flow. Should we merge them together, so that we'll have `redirect` and `popup` UX mode?

Reply:
I guess keeping them separate is not harmful.